web/themes/custom/cloudy/templates/field/field--field-efiles-link.html.twig

```diff
@@ -0,0 +1,28 @@
+{% set icon %}
+  {% include '@elements/icons/archives.twig' with {
+    "classes": [
+      "h-5",
```

Comment:
We agreed to hold off on adding Tailwind classes for a bit. There are possible conflicts between the Tailwind library and Drupal that I don't have time to unwind at the moment. Is this to set the height and width on the SVG?